src/core/friendly_errors/browser_errors.js

```diff
@@ -0,0 +1,20 @@
+// Browsers use error strings to build their error messages. These can be
+// different across different browsers. We can build a list of these strings
+// so that the FES is able to extract the required info from the errors given
+// by the browser
```

Comment:
I have a bit of trouble comprehending this comment as written. I found the explanation of this file in [this comment](https://github.com/processing/p5.js/pull/4643#issuecomment-647203541) much more clear. Perhaps a rewrite or additional explanation would be helpful.

Reply:
Okay